src/main/java/wolox/training/models/Book.java

```diff
@@ -51,6 +57,8 @@ public String getGenre() {
   }
 
   public void setGenre(String genre) {
+    Preconditions.checkNotNull(genre, CANNOT_BE_NULL, "Genre");
+    Preconditions.checkArgument(!genre.isEmpty(), CANNOT_BE_EMPTY, "Genre");
```

Comment:
You have to make the validations based on [this card](https://trello.com/c/xzO6yWKu/14-crear-modelo-de-book). As the attribute `genre` is not required, it can be null 😋

Reply:
It's done, now it can be null and empty (in case the consumer send it empty instead of null) :)